
From: Matthew Galgoci <mgalgoci@redhat.com>

This has been rejected on the grounds that ATKBD_RET_ERR is a hardware
error and needed to be printed.  But people are hitting it in real life on
non-broken hardware.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/input/keyboard/atkbd.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletion(-)

diff -puN drivers/input/keyboard/atkbd.c~disable-atykb-warning drivers/input/keyboard/atkbd.c
--- 25/drivers/input/keyboard/atkbd.c~disable-atykb-warning	2004-10-03 16:42:50.925711056 -0700
+++ 25-akpm/drivers/input/keyboard/atkbd.c	2004-10-03 16:42:50.928710600 -0700
@@ -320,7 +320,10 @@ static irqreturn_t atkbd_interrupt(struc
 			atkbd_report_key(&atkbd->dev, regs, KEY_HANJA, 3);
 			goto out;
 		case ATKBD_RET_ERR:
-			printk(KERN_DEBUG "atkbd.c: Keyboard on %s reports too many keys pressed.\n", serio->phys);
+			/*
+			 * printk(KERN_DEBUG "atkbd.c: Keyboard on %s reports "
+			 *	"too many keys pressed.\n", serio->phys);
+			 */
 			goto out;
 	}
 
_
